hypothesis-python/tests/nocover/test_integer_ranges.py

```diff
@@ -43,3 +45,16 @@ def test_intervals_shrink_to_center(inter):
         assert minimal(s, lambda x: x < center) == center - 1
     if center < upper:
         assert minimal(s, lambda x: x > center) == center + 1
+
+
+def test_bounded_integers_distribution_of_bit_width_issue_1387_regression():
+    values = []
+
+    @given(integers(0, 1e100))
+    def test(x):
+        values.append(x)
+
+    test()
+
+    huge = sum(x > 1e97 for x in values)
+    assert huge <= 0.1 * len(values)
```

Comment:
Any idea what the actual fraction is? If it's very close to `0.1` I'd be worried about this test being flaky.

Reply:
I'm *pretty* sure it's much smaller, but trying to answer has made me think that the patch has gotten too complex.  Let's see how a simpler version does...